internal/impl/cypher/output_cypher.go

```diff
@@ -77,8 +48,10 @@ func cypherOutputSpec() *service.ConfigSpec {
 			service.NewIntField(cypherMaxInFlight).
 				Description("The maximum number of queries to run in parallel.").
 				Default(64),
-			service.NewTLSToggledField(cypherTLS),
 		)
+	for _, f := range connFields() {
+		spec = spec.Field(f)
+	}
```

Comment:
nit: Why not just append these onto the spec with `Fields`?

```suggestion
		).
	  Fields(connFields()...)
```

Reply:
done